rotkehlchen/chain/evm/transactions.py

```diff
@@ -625,9 +626,26 @@ def ensure_tx_data_exists(
                 data=raw_receipt_data,
             )
 
-        tx_data = cursor.execute(query, bindings).fetchone()
```

Comment:
same question here. Why remove this query?

Reply:
same response as above. plus the only call site of `ensure_tx_data_exists()`, we're simply reconstructing the `EvmTransaction` object there.